pip_audit/_cli.py

```diff
@@ -67,7 +67,7 @@ class VulnerabilityServiceChoice(str, enum.Enum):
 
     def to_service(self, timeout: int, cache_dir: Optional[Path]) -> VulnerabilityService:
         if self is VulnerabilityServiceChoice.Osv:
-            return OsvService(timeout)
+            return OsvService(cache_dir, timeout)
```

Comment:
Yeah, I wasn't sure about that. It shouldn't cause any _problems_, but it's probably surprising behavior from `pip`'s perspective.

It might make sense to eventually unify the cache anyways as part of #71, but that's a ways away 🙂

Reply:
Okay, I refactored this so that `cache_dir` is still respected if the user explicitly supplies it, but we **never** use the `pip` cache for non-pip HTTP activity.

Reply:
Ref:

https://github.com/trailofbits/pip-audit/pull/188/files#diff-8691ff25ec6d18db328de183da5c96d97323fd9fbde5b23852b0b6ec37a75638R45-R67